src/components/Group/GroupEdit.vue

```diff
@@ -183,6 +183,10 @@ export default {
         this.$emit('save', { ...objectDiff(this.group, this.groupEdit), id: this.group.id }, event)
       }
     },
+    timezoneFilter (terms, { field, list }) {
```

Comment:
seems unrelated to the PR, does it enable more relaxed filtering of timezones? I think it's the same behavior as in the angular code, so it's good.

Reply:
Yes, I tried typing "berlin" and it didn't find `Europe/Berlin`. And yes it's unrelated, just seemed too small for the extra PR bureaucracy.